### PR TITLE
fix(ts): use explicit types in ts definitions

### PR DESCRIPTION
**Pull request by plauss**
### Context
- to fix TypeScript compilation errors due to the implicit `any` type
```
Error: node_modules/vtk.js/Sources/Common/Core/ScalarsToColors/index.d.ts:137:39 - error TS7006: Parameter 'componentIn' implicitly has an 'any' type.

137  areScalarsOpaque(scalars, colorMode, componentIn): boolean;
```
- according to [Constants.js](https://github.com/Kitware/vtk-js/blob/8511ea634a801ee50d9edbecdcf3e271c06e3916/Sources/Rendering/Core/Mapper/Constants.js) `colorMode` is of type `number`
- to be more compliant with [Kitware/VTK](https://github.com/Kitware/VTK/blob/master/Common/Core/vtkScalarsToColors.cxx#L91)

### Changes
- added explicit types for the parameters of the `areScalarsOpaque` function

### PR and Code Checklist
- [x] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [x] Run `npm run reformat` to have correctly formatted code

### CODE REVIEW

**Review by daker — APPROVED** [review]
LGTM

File: Sources/Common/Core/ScalarsToColors/index.d.ts [hunk 1]
```diff
@@ -134,7 +134,7 @@ export interface vtkScalarsToColors extends vtkObject {
 	 * otherwise return isOpaque()
 	 * @see isOpaque, getAlpha
 	 */
-	areScalarsOpaque(scalars, colorMode, componentIn): boolean;
+	areScalarsOpaque(scalars: any, colorMode: number, componentIn: number): boolean;
```
Comment (floryst) [inline Sources/Common/Core/ScalarsToColors/index.d.ts:137]: Thanks for typing this! `scalars` should be typed as `vtkDataArray`.

**Review by floryst — CHANGES_REQUESTED** [review]

File: Sources/Common/Core/ScalarsToColors/index.d.ts (hunk 1 above)
Comment (floryst) [reply Sources/Common/Core/ScalarsToColors/index.d.ts:137]: And `colorMode` should be typed as [`ColorMode`](https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Core/Mapper/Constants.d.ts#L1).

Comment (plauss) [reply Sources/Common/Core/ScalarsToColors/index.d.ts]: should be resolved by now

**Review by daker — APPROVED** [review]

**Review by floryst — APPROVED** [review]
LGTM!

**Comment by daker** [comment]
@plauss can you fix the linting issues ? https://github.com/Kitware/vtk-js/actions/runs/4829382179/jobs/8609419045?pr=2809

✖   footer's lines must not be longer than [10](https://github.com/Kitware/vtk-js/actions/runs/4829382179/jobs/8609419045?pr=2809#step:6:11)0 characters [footer-max-line-length]
⚠   footer must have leading blank line [footer-leading-blank]

**Comment by floryst** [comment]
Offending commit: [fix(ts): use explicit types](https://github.com/Kitware/vtk-js/pull/2809/commits/b213ec7e365508157a67d362abe40c971095fdfa)

I would suggest squashing your commits into one commit and ensuring that the commit body is not too long. I can't squash through Github due to limitations of their merge queue implementation.

**Comment by floryst** [comment]
Thanks @plauss!

**Pull request merged**